tdrs-backend/tdpservice/search_indexes/management/commands/clean_and_reparse.py

```diff
@@ -33,6 +33,7 @@ def add_arguments(self, parser):
         parser.add_argument("-y", "--fiscal_year", type=int, help="Reparse all files in the fiscal year, e.g. 2021.")
         parser.add_argument("-a", "--all", action='store_true', help="Clean and reparse all datafiles. If selected, "
                             "fiscal_year/quarter aren't necessary.")
+        parser.add_argument("-f", "--files", nargs='+', type=str, help="Re-parse specific datafiles by datafile id")
```

Comment:
i tested this new option on a subset of files with this command:

`python manage.py clean_and_reparse -f "1215,1030,2246,1807,1737,1826,1044"`

and this immediately triggered reparsing and did not prompt me to confirm that I wanted to reparse X number of files.

Reply:
i expected something like this:

```shell=
vcap@:~$ python manage.py clean_and_reparse -y 2025
************** reparse all False
************** selected files None

You have selected to reparse datafiles for FY 2025 and Q1-4. The reparsed files will NOT be stored in new indices and the old indices
These options will delete and reparse (2) datafiles.
Continue [y/n]?

```

Reply:
per Jan:  

> -f is only used by the action (which confirms via javascript), which is why the additional conditional checks weren't added